docs/original_alexandria_rfc.md

```diff
@@ -0,0 +1,66 @@
+## Rationale
+Documents can currently only be added to a form via file question. These do not provide search, tags, thumbnails and some other features needed for a document management service. File questions can only be used inside a form.
+
+## Goal
+Our goal is to implement an external document management service to hold and provide uploaded documents. Ideally the document management service would be independent of caluma.
```

Comment:
I think the "goal" section here is a good fragment to copy to the README to give a short introduction